library/java/io/envoyproxy/envoymobile/engine/JvmFilterContext.java

```diff
@@ -129,10 +135,11 @@ public Object onResponseTrailers(long trailerCount, long[] ignored) {
    * @param data,         buffered body data.
    * @param trailerCount, total pending trailers included in the trailer block.
    * @param endStream,    whether the stream is closed at this point.
+   * @param streamIntel,   StreamIntel not yet supported by filter callbacks.
```

Comment:
```suggestion
   * @param streamIntel,  StreamIntel not yet supported by filter callbacks.
```

Is this a valid doc string?

Reply:
Nope, it's incorrect now; I missed updating. Thanks for catching. :)